### ISSUE TITLE
Bring back the gold stars for rating

### ISSUE THREAD

**Issue by krupa** [mozilla/addons-frontend#3754]
I tried to rate an add-on and didn't realize i had rated the add-on because the stars remain grey, even after rating.

We should consider bringing back the gold stars or at least make the change obvious when users have rated an add-on.

**Comment by willdurand** [comment]
cc @pwalm

**Comment by grahamperrin** [comment]
@krupa what's the address of the page where you first observed the issue? And please, which version of the add-on did you rate?

Cross reference: https://discourse.mozilla.org/t/-/20860/19?u=grahamperrin where four or more issues coincide in a single screenshot.

**Comment by ValentinaPC** [comment]
@grahamperrin 
The gray rating stars are displayed in all of the new AMO frontend pages.
The rating stars are all the same, not matter of the add-on version.

Please see screenshots:
1. Homepage - at hover over and add-on
![at_hover](https://user-images.githubusercontent.com/15685960/32366229-e25c402e-c085-11e7-9887-d9f1c9eca5e3.png)
2. Any add-on details page in review section
![add-on_details](https://user-images.githubusercontent.com/15685960/32366277-1194ba4c-c086-11e7-962c-f36eea6fbd89.png)
3. All review page
![all_rev](https://user-images.githubusercontent.com/15685960/32366326-5180a738-c086-11e7-9d1d-14265bc51928.png)
4. etc.

**Comment by grahamperrin** [comment]
> … look disabled

Yeah, with respect: grey seems a **very** weird choice <strike>of colour </strike> to indicate that a thing has been activated, favoured, starred. Grey star? 

I truly thought that the roll-out had not yet 'brought forward' my gold star ratings. 

Shades of grey are _OK_-ish on mobile if we draw a comparison with, say, Play Store on Android but on desktop Firefox: 

- me = sad face

– should that sad face be a shade lighter, or darker, than mid-grey? ;-)

**Comment by ValentinaPC** [comment]
@grahamperrin : I was just explaining how things look like now, not expressing a personal opinion.
Jfyi I loved the golden stars, also. :)

**Comment by grahamperrin** [comment]
Thinking about this daily. Shades of grey certainly work well enough for Google's presentation of stuff in its store(s) but AMO is a very different creature. 

<https://discourse.mozilla.org/t/-/20860/43?u=grahamperrin> includes an embedded YouTube video that coincidentally captured the gold before it disappeared from the new look. 

tl;dr IMHO the gold was beautiful. 

There may be an argument to have the gold appear _only_ when hovering/pointing in the context of personas (themes/appearance add-ons) – e.g. allow the user to be _not_ distracted from the colours in the mini-previews of themes, where multiple themes are in a listing – but I'll probably respectfully argue for **consistent use of gold** (never grey) _wherever_ there's a rating.

**Comment by pwalm** [comment]
Sorry for the delay in my response:
The stars on the site are grey to reduce visual overload on the site. That being said, the only stars that should be the lovely gold colour are the stars you interact with the leave a rating:
<img width="800" alt="screen shot 2017-11-16 at 3 52 59 pm" src="https://user-images.githubusercontent.com/6797635/32922266-3e4e6834-cae6-11e7-8c48-904022c6d80e.png">

It comes down to:
- Yellow = Interactive. Personal rating.
- Grey = Not interactive. Cumulative rating.

**Comment by grahamperrin** [comment]
I agree with the wish to not overload things visually, but FWIW I don't buy into the mixed approach to star behaviours. 

Another comparison: 

![2017-11-17 13 12 54 opera add-ons](https://user-images.githubusercontent.com/192271/32949074-60260f94-cb99-11e7-9db3-4ab1fa094559.png)

![2017-11-17 13 13 12 enhancer for youtube extension - opera add-ons](https://user-images.githubusercontent.com/192271/32949081-66811bfe-cb99-11e7-97e0-22938f88a2ce.png)

![2017-11-17 13 13 26 extensions - opera add-ons](https://user-images.githubusercontent.com/192271/32949089-6b61b52a-cb99-11e7-931a-f361ef497631.png)

It might sound corny but I do 'feel the love' more where I see gold stars _from other users_.

**Comment by kumar303** [comment]
I couldn't find an issue for fixing the current stars (maybe I missed it). I filed a new one:

https://github.com/mozilla/addons-frontend/issues/4029
